tests/schema/queries/metadata.test.js

```diff
@@ -48,7 +48,7 @@ describe("Metadata query", () => {
     );
     expect(result.errors).toBeUndefined();
     expect(repositories.Metadata.findAll).toHaveBeenCalledWith({
```

Comment:
Should we add an integration test for this? Or improve the api so it isn't passing db fields in?

Reply:
I think it would be better to have an integration test.  I'll get one written.

Reply:
I feel it just is showing that the repository api is leaky - you need to know the internals.

Reply:
The best implementation would be to have an 'entity' model which would represent your data model.  This would then hide any implementation details (eg. DB naming conventions) and maintain a strict interface for us to interact with.

Reply:
I have opened a tech debt ticket to discuss. I don't think we need classes or anything like that just make the repository boundary stricter. https://trello.com/c/shtp5ndd/331-repository-api-is-leaky